requirements_dev.txt

```diff
@@ -3,6 +3,6 @@ mongomock>=4.1.2,<5
 openpyxl>=3.0.9,<3.2
 
 # Requirements for building wheel
-oldest-supported-numpy>=2023.8.3
+#oldest-supported-numpy>=2023.8.3
```

Comment:
When was this introduced and why do you remove it now just with a comment?

Reply:
It results in dependency conflicts with the numpy requirement in requirements.txt. 

I think we can remove it completely. If a normal user wants the development dependencies, there is no need to require the oldest-supported-numpy. oldest-supported-numpy should only be used internally in the docker environment where we build the wheel.